recipe_batches/recipe_batches.py

```diff
@@ -10,31 +10,23 @@ def recipe_batches(recipe, ingredients):
   i = 0
   while i < len(recipe_quantity):
     recipe_value = recipe_quantity[i]
-    print(ingredients_quantity)
     if ingredients_quantity[i] >= recipe_quantity[i]:
       ingredients_quantity[i] -= recipe_value
       if i == len(recipe_quantity) - 1:
```

Comment:
this will work fine, but it's a longer way of doing division (subtracting one at a time and keeping count of how many times you subtract.)
you could do something like `int(ingredients_qty[i]/recipe_qty[i]` or `ingredients_qty[i]//recipe_qty[i]`